src/main/webapp/themes/overview/pipe.js

```diff
@@ -315,124 +284,150 @@ function pipelineUtils() {
     }
 }
 
-function getPagination(showAvatars, component) {
-    var html = [];
-    if (!showAvatars) {
-        if (component.pagingData != '') {
-            html.push("<div class='pagination'>");
-            html.push(component.pagingData);
-            html.push("</div>");
+function addPipelineHeader(html, component, data, c, resURL) {
+    html.push('<h1>' + htmlEncode(component.name));
+    if (data.allowPipelineStart) {
+        if (component.workflowComponent) {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerBuild(\'' + component.workflowUrl + '\', \'' + data.name + '\')">');
+        } else if (component.firstJobParameterized) {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerParameterizedBuild(\'' + component.firstJobUrl + '\', \'' + data.name + '\')">');
+        } else {
+            html.push('&nbsp;<a id="startpipeline-' + c  +'" class="task-icon-link" href="#" onclick="triggerBuild(\'' + component.firstJobUrl + '\', \'' + data.name + '\')">');
         }
+        html.push('<img class="icon-clock icon-md" title="Build now" src="' + resURL + '/images/24x24/clock.png">');
+        html.push('</a>');
     }
-    return html.join("");
+    html.push('</h1>');
 }
 
-function getLink(data, link) {
-    if (data.linkRelative) {
-        return link;
+function checkDataForError(data, errrorDivId) {
```

Comment:
I would consider renaming this function to better express its intent. I wouldn't expect a function starting with the name "check" to also set values (having side effects)